.github/workflows/build.yml

```diff
@@ -140,7 +147,9 @@ jobs:
       - name: Acceptance Tests
         env:
           VAULT_TOKEN: "root"
-          VAULT_ADDR: "http://vault:8200"
+          VAULT_ADDR: "http://localhost:8200"
+          # NATIVE is true if not using jobs.<job_id>.container
+          NATIVE: "true"
```

Comment:
Want to make sure I am understanding.

1. Does this env variable act as a switch for cases where TFVP is making direct calls to the DB?
2. Does the comment mean that we are setting NATIVE to "true" because we are not running in a container?

Reply:
Yes that's right! It could also be named `NOT_DOCKERIZED` but I was trying to come up with a name that doesn't use negation.

Reply:
I think making the name more descriptive might be better. Could we flip it? `RUNS_IN_CONTIANER` and set it to false?